intelmq/tests/bots/parsers/shadowserver/test_compromised_website.py

```diff
@@ -44,6 +45,9 @@
        'time.observation': '2015-01-01T00:00:00+00:00',
        'time.source': '2017-01-16T00:43:48+00:00'}
 
+EVENT01 = copy.copy(EVENT00)
```

Comment:
@sebix EVENT00 is not an Event object. Should we convert it to Event obj before copy?